src/Elliptic/Systems/Elasticity/Equations.hpp

```diff
@@ -127,22 +127,21 @@ struct Fluxes {
 };
 
 /*!
- * \brief Compute the sources \f$S_A\f$ for the Elasticity equation.
+ * \brief Add the sources \f$S_A\f$ for the Elasticity equation.
  *
  * \see Elasticity::FirstOrderSystem
  */
 template <size_t Dim>
 struct Sources {
   using argument_tags = tmpl::list<>;
   static void apply(
```

Comment:
[optional] While the definitions are rather trivial, maybe move them into a `cpp` file? I feel like we're in the "Anything to make the compiler's life a little easier" boat :P

Reply:
Does that help for functions that are as tiny as these? Even for empty functions like those below? The drawback is that I need to duplicate the function signature and add explicit-instantiation code, so it adds a slight burden on maintainability.

Reply:
For inline functions like these it at least means the compiler doesn't have to create the function in each file, and the linker doesn't have to sort out all the weak reference. I've personally opted for doing it as a "good practice" thing, but have not measured what the impact is (partly why it's marked "optional" :P )

Reply:
Do you think it's worth adding the extra code for the function signature and the explicit instantiations in the cpp file?

Reply:
I personally do, since it's fairly easy to maintain code, but not strongly enough to force it upon anyone :)

Reply:
Moved the definitions to the cpp file